src/main/resources/com/google/api/codegen/java/standalone_sample.snip

```diff
@@ -41,42 +41,278 @@
 
 # adapted from main.snip:apiMethods(xapiClass) and its dependencies
 # For real use, we'll want to switch on the calling form id.
-@private generateSample(apiMethod, callingForm, initCode)
-  @switch callingForm
+@private generateSample(apiMethod, sample)
```

Comment:
Update the comment right above this, since we're already switching on the calling form.